bom/compile-model/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>org.eclipse.smarthome.bom.compile-model</name>
```

Comment:
`org.eclipse.smarthome` -> `org.openhab`.
But did we actually want the bom projects to be in the workspace? If not, those files should be removed.

Reply:
Sure, the BOMs provide the indices that are used by the launch configuration (bnd app).